src/main/resources/default/assets/common/common-polyfills.js.pasta

```diff
@@ -179,3 +179,21 @@ if (typeof Object.assign != 'function') {
         configurable: true
     });
 }
+
+/**@
+* Polyfill for the Object.entries function
+* see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#polyfill
+*/
+if (!Object.entries) {
+    Object.entries = function(obj) {
+        var ownProps = Object.keys(obj),
+            i = ownProps.length,
+            resArray = new Array(i); // preallocate the Array
+
+        while (i--) {
+            resArray[i] = [ownProps[i], obj[ownProps[i]]];
+        }
+
+        return resArray;
```

Comment:
```suggestion
        while (index-- > 0) {
            result[index] = [keys[index], obj[keys[index]]];
        }

        return result;
```

Reply:
`index--` and `index-- > 0' are effectively the same as the `0` is interpreted as false. Or did you suggest to change it because of readability?

Reply:
for readability